capella2polarion/converters/converter_config.py

```diff
@@ -323,13 +324,15 @@ def add_prefix(polarion_type: str, prefix: str) -> str:
 
 
 def _filter_converter_config(
-    config: dict[str, dict[str, t.Any]]
+    config: dict[str, dict[str, t.Any]],
 ) -> dict[str, dict[str, t.Any]]:
     custom_converters = (
         "include_pre_and_post_condition",
         "linked_text_as_description",
         "add_context_diagram",
         "add_tree_diagram",
+        "add_realization_diagram",
+        "add_cable_tree_diagram",
```

Comment:
Instead of adding new serializers for every diagram type, we could also add an `add_diagram` serializer and pass         `diagram_attr`, `diagram_label`, `render_params`, `filters` and `custom_field_id`. This way we could support future diagram types without adjustments in capella2polarion. At least for now I would not drop the `add_context_diagram` and `add_tree_diagram` serializers, but maybe deprecate them